src/executor/evaluate/error.rs

```diff
@@ -54,4 +54,7 @@ pub enum EvaluateError {
 
     #[error("negative substring length not allowed")]
     NegativeSubstrLenNotAllowed,
+
+    #[error("the result types should be equal: {0:?}")]
```

Comment:
```suggestion
    #[error("the result types should be equal: {0}")]
```
`:?` is Debug. Since Value is String, how about using Display?

Reply:
Good point. Thanks for the feedback 👍